### PR TITLE
Document lib/mixin/component structure #4

### PR DESCRIPTION
**Pull request by ovidiuch**
Please review @uberVU/frontend

### CODE REVIEW

File: component-mixin-lib.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+1. Strive to put as much as possible into libs.
+2. If something requires the component context, make it a mixin.
+3. If something is related to a specific UI element, put it in a component.
+
+This avoids duplication and keeps things as simple as possible.
+
+## What about CSS?
+
+This structure was designed to organize code logic best, but it can easily be
```
Comment (NiGhTTraX) [inline component-mixin-lib.md]: `...organize code logic best`? Doesn't sound right at all to me.

File: component-mixin-lib.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+adapted to styles as well. We gain more from having consistency between both
+logic and style than from optimizing the structure of the styles separately.
+
+The component domain is easy. Each component class has a corresponding style
+file with the same name. The entire style is owned under a CSS class with the
+same name, only with hyphens instead of camel case.
+
+Since we use LESS, the mixin/lib part is confusing, because they're both
```
Comment (NiGhTTraX) [inline component-mixin-lib.md:78]: Why not structure this the same way as the code section?

**Comment by ovidiuch** [comment]
@NiGhTTraX better?

I feel like there's not enough content to split style in sections as well, but the [instructions](https://github.com/uberVU/react-guide/blob/11236e0fba76d4075f9c1dfb7a0ac69cddd5f4db/component-mixin-lib.md#so-where-does-my-style-go) are now clear

**Comment by NiGhTTraX** [comment]
Looks good.

**Pull request merged**